latest/docs-ref-conceptual/tutorial-azure-cli-server.yml

```diff
@@ -0,0 +1,246 @@
+### YamlMime:YamlDocument
+documentType: Tutorial
+title: Deploy a web server in Azure
+metadata:
+  title: Deploy a web server in Azure
+  description: Get started with the Azure CLI by deploying a VM with the NGINX web server.
+  audience: IT Pro
+  level: Beginner
+  displayType: two-column
+  interactive: azurecli
+items:
+- durationInMinutes: 1
+  content: >
+    In this getting started tutorial for the Azure CLI, you will learn how to create a Linux virtual machine,
```

Comment:
Consider removing passive voice from this paragraph and perhaps add a subject to your last sentence so its not a fragment .